A62-pick-first.md

```diff
@@ -0,0 +1,189 @@
+A62: `pick_first`: sticky TRANSIENT_FAILURE and address order randomization 
+----
+* Author(s): Easwar Swaminathan (@easwars)
+* Approver: @markdroth
+* Status: Draft
+* Implemented in:
+* Last updated: 2023-04-20
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Please send an email to the grpc-io list about this gRFC and add the link to that discussion here.  Note that sending that email is what starts the 2-week clock before we can merge this gRFC.

Reply:
Done.